### PR TITLE
Switch to poetry

### PR DESCRIPTION
**Pull request by nrobinaubertin**
Applicable spec: <link>

### Checklist

- [ ] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [ ] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [ ] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [ ] The documentation is generated using `src-docs`
- [ ] The documentation for charmhub is updated
- [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

**Review by javierdelapuente — COMMENTED** [review]
is this PR still relevant?

**Review by arturo-seijas — COMMENTED** [review]
This is obviously a comment

File: .github/workflows/integration_test.yaml [hunk 1]
```diff
@@ -5,10 +5,10 @@ on:
 
 jobs:
   integration-tests:
-    uses: canonical/operator-workflows/.github/workflows/integration_test.yaml@main
```
Comment (arturo-seijas) [inline .github/workflows/integration_test.yaml:8]: Another comment

**Comment by nrobinaubertin** [comment]
Closing this as not anymore on the roadmap

**Pull request closed without merging**